src/snowcli/cli/common/flags.py

```diff
@@ -67,6 +67,26 @@
     rich_help_panel=_CONNECTION_SECTION,
 )
 
+AuthenticatorOption = typer.Option(
+    None,
+    "--authenticator",
+    help="Snowflake authenticator. Overrides the value specified for the connection.",
+    hide_input=True,
+    callback=ConnectionDetails.update_callback("authenticator"),
+    show_default=False,
+    rich_help_panel=_CONNECTION_SECTION,
+)
+
+PrivatekeypathOption = typer.Option(
+    None,
+    "--private_key_path",
```

Comment:
```suggestion
    "--private-key-path",
```
To be consistent with other options